junction/proposals/views.py

```diff
@@ -43,6 +43,11 @@ def _is_proposal_section_reviewer(user, conference, proposal):
         active=True).exists()
 
 
+def _is_proposal_author_or_proposal_reviewer(user, conference, proposal):
```

Comment:
We should move all permission check to separate file called `permissions.py` and make them public functions. Rather than naming `is_proposal_author_or_proposal_reviewer`, we can use `can_view_review_comments`, `can_review`.

We can live that for now, We will refactor code later.